osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -78,9 +78,12 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
             showNotification(@"Kalite executable not found.");
             
         } else {
+            
+            [self runKalite:@"--version"];
             NSLog([NSString stringWithFormat:@"FOUND kalite at %@!", kalite]);
             showNotification(@"KA Lite is now loaded.");
             [self getKaliteStatus];
+            
```

Comment:
@amodia Let's remove the extra blank lines and can you move the "--version" line before the "getKaliteStatus" line so that we log first then execute the command next.